openff/toolkit/utils/ambertools_wrapper.py

```diff
@@ -37,6 +37,26 @@ class AmberToolsToolkitWrapper(base_wrapper.ToolkitWrapper):
         "The AmberTools toolkit (free and open source) can be found at "
         "https://anaconda.org/conda-forge/ambertools"
     )
+    _supported_charge_methods = {
```

Comment:
(blocking) Would it be possible to leave this as `SUPPORTED_CHARGE_METHODS` to avoid an API change?

Reply:
I can maintain the API at the cost of duplicated code

Reply:
I've made an attempt to do that - unfortunately the wrappers disagree on what to call this attribute, so it's not easy to move it into the base class